packages/backend/src/open_payments/quote/service.test.ts

```diff
@@ -427,6 +434,7 @@ describe('QuoteService', (): void => {
     test('fails on unknown wallet address', async (): Promise<void> => {
```

Comment:
somewhere we should check that we are fetching a wallet address with the correct tenantId